src/System.Net.Requests/src/System/Net/WebException.cs

```diff
@@ -110,5 +111,28 @@ internal static Exception CreateCompatibleException(Exception exception)
 
             return exception;
         }
+        
+        private static WebExceptionStatus GetStatusFromExceptionHelper(HttpRequestException ex)
+        {
+            WebExceptionStatus status;
```

Comment:
Nit: this can be moved down to just before the switch

Reply:
Fixed